packages/venia-concept/src/util/makeUrl.js

```diff
@@ -38,46 +40,49 @@ const mediaBases = new Map()
  * @param {number} props.width - the desired resize width of the image
  */
 const makeOptimizedUrl = (path, { type, width } = {}) => {
-    const { href, origin } = window.location;
-    let urlObject = new URL(path, href);
+    // Immediate return if there's no image optimization to attempt
+    if (!type || !type.startsWith('image-')) {
+        return path;
+    }
+
+    const { origin } = window.location;
+    const isAbsolute = absoluteUrl.test(path);
+    let baseURL = new URL(path, mediaBackend);
 
     if (type) {
-        if (mediaBases.has(type)) {
+        // If URL is relative and has a supported type, prepend media base onto path
+        if (!isAbsolute && mediaBases.has(type)) {
```

Comment:
I made the decision to remove functionality that tries to inject media bases into absolute URLs. Logically, I couldn't think of a use case where we receive an absolute URL that doesn't already have the media base in it. If I'm missing something, let me know, and I will re-implement.